src/main/java/duke/datetime/DateTime.java

```diff
@@ -0,0 +1,151 @@
+package duke.datetime;
+
+import duke.exception.DukeException;
+
+/**
+ * Enables Duke to understand date and times.
+ */
+public class DateTime {
+
+    private int year;
+    private int month;
+    private int day;
+    private int time;
+
+    /**
+     * Initialises appropriate date and time fields for Duke to understand.
+     *
+     * @param year 4-digit integer.
+     * @param month Integer value of the month.
+     * @param day Integer value of the day of the month.
+     * @param time 4-digit integer number (time in 24 hour format).
+     */
+    public DateTime(int year, int month, int day, int time) {
+        this.year = year;
+        this.month = month;
+        this.day = day;
+        this.time = time;
+    }
+
+    /**
+     * Creates a DateTime object by splitting up the date and time given in a string.
+     *
+     * @param dateTimeString Date and time given in the format DD/MM/YYYY HHMM.
+     * @return DateTime object created.
+     * @throws ArrayIndexOutOfBoundsException If date and time given is not in the accepted format.
+     */
+    public static DateTime create(String dateTimeString) throws ArrayIndexOutOfBoundsException {
```

Comment:
If you want to make the code more foolproof, you can throw an exception when you encounter 

1. negative date / negative time

1.  if time and day and/or month are out of allowed values. Using Calendar class might help in some of these.